website/app/Http/Controllers/EventsController.php

```diff
@@ -48,9 +48,9 @@ public function store(CreateEventRequest $request)
       $date     = $date === null ? null : new Carbon($date);
       
       Event::create(compact('title', 'date', 'location'));
-      return;
+      return response()->json("success" => true);
     }
-    return redirect(action('EventsController@index'));
+    return response()->json("success" => false);
```

Comment:
So `"success" => false` is correct but you are trying to declare an associative array. This is the incorrect syntax for it. It should be `response()->json(array("success" => false))`